mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -584,7 +584,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 3 damageSource
 	METHOD method_5748 getMaxAir ()I
 	METHOD method_5749 readCustomDataFromNbt (Lnet/minecraft/class_2487;)V
```

Comment:
Something like `readCustomDataNbt` per current convention.

But before we make the change, Imo we need a more concise name if we do rename this. This is used literally by every mod, and we definitely need a very concise name (needs debate!)

Reply:
`readAdditionalNbt`? Might be a bit obscure, but imo this mostly need docs.

Reply:
This probably deserves a separate discussion/debate, maybe notify on discord and talk in a github issue (stuff get flushed out on discord fast while no one look at github issues)

Reply:
Notified Discord and opened #2259.